definitions/procedures/backup/config_files.rb

```diff
@@ -17,12 +17,15 @@ class ConfigFiles < ForemanMaintain::Procedure
             :array => true, :default => ['all']
       param :ignore_changed_files, 'Should packing tar ignore changed files',
             :flag => true, :default => false
+      param :exclude_qpidd_data, 'Exclude qpidd /var/lib/qpidd/ Directory during backup',
+            :flag => true, :default => false
     end
 
     # rubocop:disable Metrics/MethodLength
     def run
       logger.debug("Invoking tar from #{FileUtils.pwd}")
-      tar_cmd = tar_command
+      configs, to_exclude = config_files
```

Comment:
Instead of passing configs and to_exclude to `tar_command` method, I think it would be good to have `options` to tar_command method which will accept extra directories to exclude and append to_exclude list inside tar_command?